generate_sample_file.py

```diff
@@ -205,7 +210,8 @@ def _write_row(self, writer, sequential_arid, treatment_code, address_type, expe
             'TREATMENT_CODE': treatment_code["treatment_code"],
             'FIELDCOORDINATOR_ID': '',
             'FIELDOFFICER_ID': '',
-            'CE_EXPECTED_CAPACITY': expected_capacity
+            'CE_EXPECTED_CAPACITY': expected_capacity,
+            'CE_SECURE': self.get_random_ce_secure()
```

Comment:
I'd prefer this to be tied to when we Create CE and SPG only to create more realistic data.

Reply:
That's a massive "nice to have". This random sample doesn't even create any random CE cases yet. We should just change it to zero, and not have it random if you're going to fuss.

Reply:
The sample does create random CE cases (line 147) but an if statement has now been included to randomly assign a 0/1 value in CE_SECURE is the address_type is 'CE'